config/initializers/geocoder.rb

```diff
@@ -0,0 +1,21 @@
+Geocoder.configure(
+  # geocoding options
+  # :timeout      => 3,           # geocoding service timeout (secs)
+  lookup: :google,     # name of geocoding service (symbol)
+  # :language     => :en,         # ISO-639 language code
+  use_https: true,       # use HTTPS for lookup requests? (if supported)
+  # :http_proxy   => nil,         # HTTP proxy server (user:pass@host:port)
+  # :https_proxy  => nil,         # HTTPS proxy server (user:pass@host:port)
+  :api_key      => 'AIzaSyCwj-gjri-79xhDGM5bYpt6lYKbMTzzxIs'         # API key for geocoding service
```

Comment:
Line is too long. [100/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.